<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>JavaScript Exception</title>
    <script src="helper.js"></script>
</head>
<body>
<ol>
    <li>
        普通异常处理。<br/>
        <kbd>
            <pre>try{
    alert(parseInt(age));
}catch (ex){
    alert('异常消息：' + ex.message);
}finally {
    alert('异常处理完毕。');
}</pre>
        </kbd>
    </li>
    <li>
        抛出异常并处理。<br/>
        <kbd>
            <pre>try{
    throw new Error('Whoops!');
}catch (ex){
    alert('异常消息：'+ex.message);
}</pre>
        </kbd>
    </li>
    <li>
        不处理抛出的异常，程序无法继续执行。<br/>
        <kbd>
            throw new Error('Whoops...');<br/>
            console.log('程序运行完毕。'); //连续运行这两句
        </kbd>
    </li>
</ol>
</body>
</html>